### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by davidgfnet** [JiriBilek/WiFiSpiESP#1]
Just importing it and building as is raises some errors:

/WiFiSPIESP.ino: In function 'void setup()':
WiFiSPIESP:58: error: 'class WiFiClass' has no member named 'persistent'
     WiFi.persistent(false);  // Solving trap in ESP8266WiFiSTA.cpp#144 (wifi_station_ap_number_set)
          ^
exit status 1
'class WiFiClass' has no member named 'persistent'

It's not just one but a bunch of them that come later.

**Comment by JiriBilek** [comment]
Hi,
I have just compiled it with no problem.
My environment is Arduino IDE 1.8.5 and esp8266 toolchain 2.3.0

Are you sure you switched your board in Arduino IDE to 8266? E.g. NodeMCU or Generic 8266 should work.

**Comment by davidgfnet** [comment]
I'm pretty sure I did.
There's some linker errors too, I think due to stuff being defined in headers without static and the linker being able to find multiple defintions of them.
I actually happened to write the same thing then to find your project later. I did not write it in arduino though. Still does not work well so far, since it seems that the SPI slave is not working yet, due to ISR not being called. I'm not using the SS pin and I disabled it, maybe the technical reference is wrong and it is still active :( Friking ESP8266 and dodgy chiniese docs...
